bot/grenouille_http_server.py

```diff
@@ -0,0 +1,50 @@
+from http.server import BaseHTTPRequestHandler, HTTPServer
+import threading
+import logging
+import cgi
+import os
+
+
+def MakeHandler(bot):
+    class testHTTPServer_RequestHandler(BaseHTTPRequestHandler):
+        
+        def do_POST(self):
+            form = cgi.FieldStorage(
+                fp=self.rfile,
+                headers=self.headers,
+                environ={"REQUEST_METHOD": "POST"}
+            )
+            
+            for item in form.list:
+                if (item.name == "say"):
+                    bot.send_msg(item.value)
+                    
+                    self.send_response(200)
+                    self.send_header("Content-type", "text/html")
+                    self.end_headers()
+                    return
+    return testHTTPServer_RequestHandler
+
+class GrenouilleHttpServer:
```

Comment:
Something like:

```
from threading import Thread

class GrenouilleHttpServer(Thread):
   def run(self, grenouille_bot):
      self.bot = grenouille_bot
      port = int(os.environ['WEBSERVER_PORT'])
      server_address = ('127.0.0.1', port)
      myhandler = MakeHandler(self.bot)
      httpd = HTTPServer(server_address, myhandler)
      logging.info('webserver started on port ' + str(port))
      httpd.serve_forever()
```

After you add `self.grenouille_http_server.start()` in the main class. This way you don't have to cross thread starting by calling it in `grenouille_irc_bot.py`